src/Workspaces/Core/Portable/Rename/RenameUtilities.cs

```diff
@@ -109,7 +109,8 @@ internal static IEnumerable<Document> GetDocumentsAffectedByRename(ISymbol symbo
         private static bool ShouldRenameOnlyAffectDeclaringProject(ISymbol symbol)
         {
             // Explicit interface implementations can cascade to other projects
-            return symbol.DeclaredAccessibility == Accessibility.Private && !symbol.ExplicitInterfaceImplementations().Any();
+            // check if the symbol is override. https://github.com/dotnet/roslyn/issues/25682
```

Comment:
Agreeing with @CyrusNajmabadi: the comment here might be better stated "we check if the symbol is an override. If it's overriding something (even if private) it's an error, but we can rename across that error."

Reply:
>  If it's overriding something (even if private) it's an error, but we can rename across that error."

If we want to rename across accessibility errors, we should likely just remove this method entirely.  This fix here will only address one case where such referencing illegally occurs.  But you can run into this as well without needing overrides.

Reply:
@JieCarolHu That seems reasonable.  However, it's unclear ot me why we're only checking .IsOverride?  What aobut the same scenario, but with an abstract method, or a virtual method?